s2i-datapower/README.md

```diff
@@ -0,0 +1,119 @@
+# Datapower s2i
+
+This project demonstrates how to create an s2i Datapower image to facilitate Datapower deployments in Openshift.
+
+![IBM Datapower](https://avatars3.githubusercontent.com/u/8836442?v=3&s=200 "IBM Datapower")
+
+## Table of content
+
+* [Create a Project ](#create-a-project)
+* [Create the s2i-datapower Image](#create-the-s2i-datapower-image)
+* [The Experimentation Use Case](#the-experimentation-use-case)
+* [The Immutable Use Case](#the-immutable-use-case)
+* [Examples](#examples)
+
+
+
+## Create a Project 
+
+This is the project win which we will run the below commands, you can use a different project.
+
+```
+oc new-project datapower
+```
+
+## Create the s2i-datapower Image
+
+This commands will create a build config whose outout will be the s2i-datapower.
+
+```
+oc new-build https://github.com/raffaelespazzoli/containers-quickstarts#datapower --context-dir=s2i-datapower --name=s2i-datapower --strategy=docker
+```
+The s2i-datapower image supports two use cases:
+
+* experimentation 
+* immutable
+
+## The Experimentation Use Case
+
+In this use case the user connects to the datapower instance and creates the configurations using the web ui.
+
+The configurations are save in two persistent storage volumes (/drouter/config and /drouter/local)
+
+The expectation is that the configurations can be retrieved from the persistent volumes and use as an input for the s2i-datapower image in the immutable use case.
+
+This is just one possbile approach to creating configuration. Any approach is fine.
+
+to use the image in this mode issue the following commands
+
+```
+oc new-app --docker-image=datapower/s2i-datapower -e DATAPOWER_ACCEPT_LICENSE=true -e DATAPOWER_WORKER_THREADS=4 --name=datapower
+oc set volume dc/datapower --add -m /drouter/config --name datapower-config -t pvc --claim-name=datapower-config --claim-size=1G
```

Comment:
fixed, I think is has to do with the fact that I did this project a while ago months ago.